presto-orc/src/test/java/com/facebook/presto/orc/AbstractTestOrcReader.java

```diff
@@ -207,7 +208,14 @@ private void testRoundTripNumeric(Iterable<? extends Number> values)
         tester.testRoundTrip(
                 javaTimestampObjectInspector,
                 writeValues.stream()
-                        .map(timestamp -> new SqlTimestamp(timestamp, UTC_KEY))
+                        .map(timestamp -> {
+                            if (SESSION.isLegacyTimestamp()) {
```

Comment:
You know `SESSION`  is has `UTC` time zone key, so you could simply `sqlTimestampOf(timestamp, SESSION)`, right?